index.js

```diff
@@ -53,6 +53,21 @@ const pkg = require('./package.json');
  *     output: 'css/file.css',
  *     sourceMap: true
  * }
+ * 
+ * @example
+ * // compiles multiple files and writes all to single file
+ * {
+ *     file: 'src/*.scss',
+ *     output: 'main.css'
+ * }
+ * 
+ * @example
+ * // compile compressed css and write to file with `.min` suffix
+ * {
+ *     file: 'src/*.scss',
+ *     output: srcFile => srcFile.replace(/.scss$/, '.min.css'),
```

Comment:
@phillipluther After adding this, it made me really want to add #16 because this is really clunky. If I want to change the file extension and the directory I would have to have to replace methods. Whereas, with interpolaters it would be much slicker: `output: 'css/%n.min.css'`.

Reply:
I feel that. Though I'd also throw that into a "sweet, but not essential feature" for release. I vote we create a new tag for `next-release` or `[some-targeted-version]` and include this one in the next publish.

Reply:
Agreed. I actually wasn't suggesting we throw it in before the initial release, just calling out that it's a feature I really want to add.